test/Templates.Test/Helpers/TemplatePackageInstaller.cs

```diff
@@ -72,13 +73,13 @@ private static void ReinstallTemplatePackages(ITestOutputHelper output)
         private static void VerifyCannotFindTemplate(ITestOutputHelper output, string templateName)
         {
             // Verify we really did remove the previous templates
-            var tempDir = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString("D"));
+            var tempDir = Path.Combine(AppContext.BaseDirectory, Path.GetRandomFileName(), Guid.NewGuid().ToString("D"));
```

Comment:
Out of interest, why do you want the `tempDir` to be inside `AppContext.BaseDirectory`?

Reply:
global.json. On my machine, I have a bunch of 2.1.0 stuff installed. When running without a global.json file, the tests pick up the latest installed dotnet-sdk. It caused failures because `dotnet new` without `--framework-override` would default to the netcoreapp2.1 templates.

Reply:
Makes sense. Thanks.